install_config/provisioners.adoc

```diff
@@ -55,21 +54,24 @@ which the `install` variable is `true`.
 |If `true`, deploy all provisioners that have their respective `install` variables set as `true`, otherwise, remove them.
 
 |`openshift_provisioners_image_prefix`
-|The prefix for the component images. For example, with
+|The prefix for the component images.
 ifdef::openshift-origin[]
-`openshift/origin-efs-provisioner:v1.0.0`, set prefix `openshift/origin-`.
+For example, with `openshift/origin-efs-provisioner:v1.0.0`, set prefix
+`openshift/origin-`.
 endif::[]
 ifdef::openshift-enterprise[]
-`openshift3/efs-provisioner:v3.6`, set prefix `openshift3/`.
+Defaults to `registry.access.redhat.com/openshift3/`, set it to a different
+value if you are using an alternative registry.
```

Comment:
I have updated this for _enterprise_, are there any changes for _origin_?

Reply:
Sorry, have no idea about Origin

Reply:
Thank you @gpei Are these changes only applicable to v 3.9? what about 3.5, 3.6, and 3.7?